### ISSUE TITLE
Got 'undefined' when using angular-translate directive

### ISSUE THREAD

**Issue by Uxspy** [angular-translate/angular-translate#971]
Hi, we've upgrade angular-translate from 2.5.2 to 2.6.1 and found regression issues when using translate directives. Below is the html templates:

```html
<div translate>{{label}}</div>
````

When $scope.label is **undefined**, in angular-translate 2.5, the translated result is:

```html
<div></div>
```

However in angular-translate 2.6, the result is:

```html
<div>undefined</div>
```

Could anyone tell me is this a change or this is a bug? Thanks!

**Comment by tspaeth** [comment]
@knalli due to the #854 fix on #701 - https://github.com/angular-translate/angular-translate/blob/master/src/directive/translate.js#L147-L150

On possible fix is to extend the updateTranslations to check for undefined... but does it have any other consequences?

```
if (translationIds.hasOwnProperty(key) && translationIds[key] !== undefined) {
```

**Comment by knalli** [comment]
:shipit: :)

**Comment by knalli** [comment]
Fix landed as 3f7cf4c

**Comment by Uxspy** [comment]
Thanks, that's Great!
